examples/demo_workflow/app.py

```diff
@@ -106,7 +107,7 @@ def act_for_child_wf(ctx: WorkflowActivityContext, inp):
 
 
 def main():
-    with DaprClient() as d:
+    with DaprWorkflowClient() as wfc:
```

Comment:
This is changing what this example (which also runs as an integration test) does!

The point was that you can use the regular Dapr SDK to start a workflow as well - you do not have to use the workflow SDK for that. We will want to retain this.

Reply:
this is equivalent to https://github.com/dapr/dotnet-sdk/issues/1344 and issues like this have been created in all other SDKs, the workflow functions in the `DaprClient` are now deprecated and we shouldn't advice users to use them. Thats why changing the examples.

Reply:
If we're deprecating workflows in `DaprClient` we should move this example to the https://github.com/dapr/python-sdk/tree/main/examples/workflow directory where the other `dapr-ext-workflow` examples live. We should also check for any overlap with what the existing tests are checking.

Reply:
the point of changing this is that we no longer want to encourage users to use the `DaprClient` to interact with Workflows.

examples usually are the first point of contact for users, so intentionally leaving a deprecated example just because it acts as a test is a mistake in my opinion.

Reply:
but since we already have examples of the correct SDK to use here https://github.com/dapr/python-sdk/tree/main/examples/workflow

I'll revert the changes made to this file

Reply:
@famarting @elena-kolevska will the [Python workflow quickstart](https://github.com/dapr/quickstarts/blob/release-1.14/workflows/python/sdk/order-processor/app.py) also be updated to `dapr-ext-workflow` instead of `DaprClient`?

Reply:
wow, I didnt realize of that, well, all quickstarts in that repo using the equivalent to the DaprClient will also need to be updated

Reply:
Update, the quickstarts were updated [here](https://github.com/dapr/quickstarts/pull/1101).